README.md

```diff
@@ -32,6 +32,20 @@ via Middleman.
 To run a development server just use the command:
 `middleman server`
 
+## Testing
+Unit Tests use Karma and Chai.
+
+To run tests:
+
+- Ensure node dependencies are installed with `npm install`.
+- Run `npm run test:unit`.
+- Karma will instruct you to open a browser to run the tests.
+
+Snapshot tests use Selenium
```

Comment:
@danivovich @sdroadie Cool, you guys noticed this and added to the documentation. Thanks!